### PR TITLE
New version: OSGeo.QGIS version 3.22.3

### LINKED ISSUE #41348: [Package Request]: OSGeo.QGIS
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not a standalone or portable executable or a .zip file
- [X] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: QGIS Development Team
Publisher Url: https://www.osgeo.org
Package Name: QGIS
Description: QGIS is a user-friendly Open Source Geographic Information System (GIS) licensed under the GNU General Public License. QGIS is an official project of the Open Source Geospatial Foundation (OSGeo). It runs on Linux, Unix, Mac OSX, Windows and Android and supports numerous vector, raster, and database formats and functionalities.
Package Version: 3.22.3
Installer URL: https://qgis.org/downloads/QGIS-OSGeo4W-3.22.3-1.msi

Current Package Identifier (for new versions): OSGeo.QGIS

### PR DESCRIPTION
**Pull request by Trenly**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

* Resolves #41348 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41360)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
Might want to add `Resolves #41348 =))`
soon done with the forever download

File: manifests/o/OSGeo/QGIS/3.22.3/OSGeo.QGIS.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS1.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: OSGeo.QGIS
+PackageVersion: 3.22.3
+PackageLocale: en-US
+Publisher: QGIS Development Team
```
Comment (OfficialEsco) [inline manifests/o/OSGeo/QGIS/3.22.3/OSGeo.QGIS.locale.en-US.yaml:7]: Welp there is also this
```suggestion
Publisher: QGIS.org
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
> Might want to add `Resolves #41348` =)) soon done with the forever download

I just assumed the previous metadata was correct. Silly me. My bandwidth doesn't like those big files

**Review by OfficialEsco — APPROVED** [review]
> > Might want to add `Resolves #41348` =)) soon done with the forever download
> 
> I just assumed the previous metadata was correct. Silly me. My bandwidth doesn't like those big files

It was probably wrong before tbh
Also its the webpage.. think it took 30-40 min 😴

**Pull request merged**